File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.cs
using System;

class Solution {
    public String solution(String rsp) {
        String answer = "";
        for (int i = 0; i < rsp.Length; i++) {
            char ch = rsp[i];
            if (ch == '2') {
                answer += '0';
            } else if (ch == '0') {
                answer += '5';
            } else if (ch == '5') {
                answer += '2';
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.py
def solution(rsp):
    answer = "";
    for i in range(len(rsp)):
        ch = rsp[i]
        if (ch == '2') :
            answer += '0'
        elif (ch == '0') :
            answer += '5'
        elif (ch == '5') :
                answer += '2'
            
        
    return answer;